target/linux/ramips/image/mt7621.mk

```diff
@@ -2551,6 +2551,8 @@ TARGET_DEVICES += tplink_ec330-g5u-v1
 
 define Device/tplink_er605-v2
   $(Device/nand)
+  DEVICE_COMPAT_VERSION := 2.0
+  DEVICE_COMPAT_MESSAGE := Config cannot be migrated because interface names have changed
```

Comment:
You also need to increase version number in:
https://github.com/openwrt/openwrt/blob/06b37a5856ac7d0a2ddc2c0745ac1da3a01688d6/target/linux/ramips/mt7621/base-files/etc/board.d/05_compat-version#L11

Reply:
Updated, thanks!